spec/spec_helper.rb

```diff
@@ -1,8 +1,62 @@
-require 'coveralls'
-require 'simplecov'
+ENV['RACK_ENV'] = 'test'
+
+require File.join(File.dirname(__FILE__), '..', 'lib/app.rb')
+
+require 'byebug'
+require 'capybara'
+require 'capybara/rspec'
+require 'database_cleaner'
+require 'dm-rspec'
+require 'pry'
+require 'rspec'
+require 'tilt/erb'
+require 'coveralls' # Makers
+require 'simplecov' # Makers
 
 SimpleCov.formatters = [
-  SimpleCov::Formatter::HTMLFormatter,
-  Coveralls::SimpleCov::Formatter
+    SimpleCov::Formatter::HTMLFormatter,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.